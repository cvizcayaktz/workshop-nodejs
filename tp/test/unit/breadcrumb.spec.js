const React = require('react');
const renderer = require('react-test-renderer');
const Breadcrumb = require('./views/components/Breadcrumb.jsx');

describe('Breadcrumb (Snapshot)', () => {
  const component = renderer.create(<Breadcrumb />);

  it('Should only render home link', () => {

  });
});
