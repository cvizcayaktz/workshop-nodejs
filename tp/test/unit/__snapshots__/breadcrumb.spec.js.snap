// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb (Snapshot) Breadcrumb renders with results 1`] = `
<ul
  class="breadcrumb"
>
  <li
    class="breadcrumb__element"
  >
    <a
      href="/"
    >
      Inicio
    </a>
  </li>
  <li
    className="breadcrumb__element"
  >
    cat 1
  </li>
  <li
    className="breadcrumb__element"
  >
    cat 2
  </li>
  <li
    className="breadcrumb__element"
  >
    cat 3
  </li>
  <li
    className="breadcrumb__element"
  >
    cat 4
  </li>
  
</ul>
`;

exports[`Breadcrumb (Snapshot) Breadcrumb renders without results 1`] = `
<ul
  class="breadcrumb"
>
  <li
    class="breadcrumb__element"
  >
    <a
      href="/"
    >
      Inicio
    </a>
  </li>
  
  
</ul>
`;
